### PR TITLE
Stabilize packages for 1.1.0 RTM

### PR DESCRIPTION
**Pull request by gkhanna79**
@weshaggard @dagood @schellap @ericstj PTAL

@mellinoe PTAL to confirm the new distro list in stable.packages.props is complete.

CC @leecow

### CODE REVIEW

**Comment by gkhanna79** [comment]
@dagood Will Maestro perform stabilized version update in https://github.com/dotnet/core-setup/blob/release/1.1.0/pkg/projects/Microsoft.NETCore.App/project.json and https://github.com/dotnet/core-setup/blob/release/1.1.0/build_projects/shared-build-targets-utils/DependencyVersions.cs when corresponding CoreFX and CoreCLR stabilized version packages are published?

File: pkg/stable.packages.props [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  <ItemGroup>
+
+    <!-- NETCore 1.1.0 core-setup -->
+    <StablePackage Include="Microsoft.NETCore.App">
+      <Version>1.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore">
+      <Version>5.0.3</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.UniversalWindowsPlatform">
```
Comment (weshaggard) [inline pkg/stable.packages.props]: The only meta-package that we want to mark stable is Microsoft.NETCore.App. We don't want to marke the others as stable.

**Comment by gkhanna79** [comment]
Also updating https://github.com/dotnet/core-setup/blob/release/1.1.0/build_projects/shared-build-targets-utils/Utils/HostVersion.cs#L66 that got missed in the initial commit. Testing this right now.

File: pkg/stable.packages.props (hunk 1 above)
Comment (gkhanna79) [inline pkg/stable.packages.props]: Fixed.

**Comment by dagood** [comment]
The Maestro auto-update should change those versions from prerelease to stable, yes.

File: pkg/stable.packages.props [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    <StablePackage Include="runtime.ubuntu.16.04-x64.Microsoft.NETCore.DotNetHost">
+      <Version>1.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.Microsoft.NETCore.DotNetHostPolicy">
+      <Version>1.1.0</Version>
+    </StablePackage>
+    <StablePackage Include="runtime.ubuntu.16.04-x64.Microsoft.NETCore.DotNetHostResolver">
+      <Version>1.1.0</Version>
+    </StablePackage>
+     <StablePackage Include="runtime.ubuntu.16.10-x64.Microsoft.NETCore.DotNetHost">
```
Comment (mellinoe) [inline pkg/stable.packages.props]: very minor nit: extra space here

File: pkg/projects/packages.builds [hunk 3]
```diff
@@ -2,9 +2,16 @@
...
-  <ItemGroup>
+  <PropertyGroup>
+    <BuildAllPackages>true</BuildAllPackages>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'true'" >
     <Project Include="*\*.builds" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false'" >
```
Comment (mellinoe) [inline pkg/projects/packages.builds]: What is this?

Comment (gkhanna79) [inline pkg/projects/packages.builds]: This is about adding the infrastructure that we will use during servicing builds from this branch post RTM.

**Comment by mellinoe** [comment]
Distro list looks correct to me

**Comment by gkhanna79** [comment]
@schellap Can you please review the changes?

File: build_projects/shared-build-targets-utils/Utils/BuildVersion.cs [hunk 4]
```diff
@@ -7,7 +7,7 @@ public class BuildVersion : Version
         public string SimpleVersion => $"{Major}.{Minor}.{Patch}.{CommitCountString}";
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
```
Comment (schellap) [inline build_projects/shared-build-targets-utils/Utils/BuildVersion.cs]: Only the NuGetVersion part.

**Comment by schellap** [comment]
LGTM. Can merge once green.

**Pull request merged**